### DISCUSSION TITLE
Lora node

### DISCUSSION THREAD

**Discussion by Wundrwaffle** [ModelSurge/sd-webui-comfyui#88] [Q&A] [answered]
Hello. MAybe not only Lora, but i have a question. 
What strength_clip and strength_model do? Kind of strength of lora and clip skip of model? 
![Screenshot_2](https://github.com/ModelSurge/sd-webui-comfyui/assets/117706507/c2347769-3c8a-4cb5-be5f-ecf2a2537d39)

At webui there are only strength of lora
![Screenshot_3](https://github.com/ModelSurge/sd-webui-comfyui/assets/117706507/71948a96-a0ff-426b-b3fe-f74beb212d49)

**Comment by PladsElsker** [comment]
Hi again. 

The Load LoRa node is not from this repo. 

The additional networks extension is not linked to that node. They are independant. 

I never really used either of these, I usually just put the LoRA network in my prompt in A1111. 

If you have specific questions about the node or the additional network extension, I would advise asking in their respective repository.

**Reply by PladsElsker** [reply]
If you just want to know about what they do, they apply strength to the LoRA being applied. 1 gives full strength to the LoRA, while 0 means tbat the LoRA won't do anything to the diffusion process.

**Reply by PladsElsker** [reply, marked as answer]
Strength Clip would be for the text encoding/decoding part of the LoRA, while the other woud be for the unet strength. So one is more focused on affecting how the prompt is percieved, while the other one affects more how the percieved prompt is being diffused. 

This is a really oversimplified view of what's happenning, but hopefully it gets the idea accross.
